notebook/services/api/api.yaml

```diff
@@ -543,6 +549,73 @@ paths:
           description: Configuration object
           schema:
             type: object
+            items:
+              $ref: '#/parameters/section_name'
+
+  /terminals:
+    get:
+      summary: Get available terminals
+      tags:
+        - terminals
+      responses:
+        200:
+          description: A list of all available terminal ids.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Terminal_ID'
+        403:
+          description: Forbidden to access
+        404:
+          description: Not found
+
+    post:
+      summary: Create a new terminal
+      tags:
+        - terminals
+      responses:
+        200:
+          description: Response description
+          schema:
+            $ref: '#/definitions/Terminal_ID'
+        403:
+          description: Forbidden to access
+        404:
+          description: Not found
+
+  /terminals/{terminal_id}:
+    get:
+      summary: Get a terminal session corresponding to an id.
+      tags:
+        - terminals
+      parameters:
+        - $ref: '#/parameters/terminal_id'
+      responses:
+        200:
+          description: An object containing
+          schema:
+            $ref: '#/definitions/Terminal_ID'
+        403:
+          description: Forbidden to access
+        404:
+          description: Not found
+
+    delete:
+      summary: Delete a terminal session corresponding to an id.
+      tags:
+        - terminals
+      parameters:
+        - $ref: '#/parameters/terminal_id'
+      responses:
+        204:
+          description: Response description
```

Comment:
This looks like placeholder text that should be replaced with an actual description